.travis.yml

```diff
@@ -14,7 +14,7 @@ jdk:
   - oraclejdk8
 
 script:
-  - ./gradlew clean check -x checkstyleTest --stacktrace
+  - ./gradlew clean check -x checkstyleTest -x :apollo-gradle-plugin:test --stacktrace
```

Comment:
Sorry I meant in general CI should be running Build task rather than Check test. If you only run check you leave possibility of the check passing but archiving failing due to packaging issue